sdk/v2_workflow_run.go

```diff
@@ -34,24 +34,24 @@ type V2WorkflowRunHookRequest struct {
 }
 
 type V2WorkflowRun struct {
-	ID            string                 `json:"id" db:"id" cli:"id"`
-	ProjectKey    string                 `json:"project_key" db:"project_key"`
+	ID            string                 `json:"id" db:"id" cli:"id" action_metadata:"workflow-run-id"`
+	ProjectKey    string                 `json:"project_key" db:"project_key" action_metadata:"project-key"`
 	VCSServerID   string                 `json:"vcs_server_id" db:"vcs_server_id"`
-	VCSServer     string                 `json:"vcs_server" db:"vcs_server"`
+	VCSServer     string                 `json:"vcs_server" db:"vcs_server" action_metadata:"vcs-server"`
 	RepositoryID  string                 `json:"repository_id" db:"repository_id"`
-	Repository    string                 `json:"repository" db:"repository"`
-	WorkflowName  string                 `json:"workflow_name" db:"workflow_name" cli:"workflow_name"`
+	Repository    string                 `json:"repository" db:"repository" action_metadata:"repository-identifier"`
```

Comment:
We should match with existing log field no?
https://github.com/ovh/cds/blob/master/sdk/log/fields.go#L37

Reply:
But this one is used for router logs: https://github.com/ovh/cds/blob/master/sdk/doc/route.go#L93